---
page: jamboree
navText: HTAN Data Jamboree
---

<meta content="text/html; charset=UTF-7" http-equiv="content-type" /><style
    type="text/css"
>
    @import url(https://themes.googleusercontent.com/fonts/css?kit=fpjTOVmNbO4Lz34iLyptLUXza5VhXqVC6o75Eld_V98);
    ol.lst-kix_list_1-3 {
        list-style-type: none;
    }
    ol.lst-kix_list_1-4 {
        list-style-type: none;
    }
    .lst-kix_list_2-6 > li:before {
        content: '' counter(lst-ctn-kix_list_2-6, decimal) '. ';
    }
    .lst-kix_list_2-7 > li:before {
        content: '' counter(lst-ctn-kix_list_2-7, lower-latin) '. ';
    }
    .lst-kix_list_2-7 > li {
        counter-increment: lst-ctn-kix_list_2-7;
    }
    ol.lst-kix_list_1-5 {
        list-style-type: none;
    }
    ol.lst-kix_list_1-6 {
        list-style-type: none;
    }
    .lst-kix_list_2-1 > li {
        counter-increment: lst-ctn-kix_list_2-1;
    }
    ol.lst-kix_list_1-0 {
        list-style-type: none;
    }
    .lst-kix_list_2-4 > li:before {
        content: '' counter(lst-ctn-kix_list_2-4, lower-latin) '. ';
    }
    .lst-kix_list_2-5 > li:before {
        content: '' counter(lst-ctn-kix_list_2-5, lower-roman) '. ';
    }
    .lst-kix_list_2-8 > li:before {
        content: '' counter(lst-ctn-kix_list_2-8, lower-roman) '. ';
    }
    ol.lst-kix_list_1-1 {
        list-style-type: none;
    }
    ol.lst-kix_list_1-2 {
        list-style-type: none;
    }
    .lst-kix_list_1-1 > li {
        counter-increment: lst-ctn-kix_list_1-1;
    }
    ol.lst-kix_list_2-6.start {
        counter-reset: lst-ctn-kix_list_2-6 0;
    }
    ol.lst-kix_list_1-8.start {
        counter-reset: lst-ctn-kix_list_1-8 0;
    }
    ol.lst-kix_list_2-3.start {
        counter-reset: lst-ctn-kix_list_2-3 0;
    }
    ol.lst-kix_list_1-5.start {
        counter-reset: lst-ctn-kix_list_1-5 0;
    }
    ol.lst-kix_list_1-7 {
        list-style-type: none;
    }
    .lst-kix_list_1-7 > li {
        counter-increment: lst-ctn-kix_list_1-7;
    }
    ol.lst-kix_list_1-8 {
        list-style-type: none;
    }
    ol.lst-kix_list_2-5.start {
        counter-reset: lst-ctn-kix_list_2-5 0;
    }
    .lst-kix_list_2-0 > li {
        counter-increment: lst-ctn-kix_list_2-0;
    }
    .lst-kix_list_2-3 > li {
        counter-increment: lst-ctn-kix_list_2-3;
    }
    .lst-kix_list_2-6 > li {
        counter-increment: lst-ctn-kix_list_2-6;
    }
    ol.lst-kix_list_1-7.start {
        counter-reset: lst-ctn-kix_list_1-7 0;
    }
    .lst-kix_list_1-2 > li {
        counter-increment: lst-ctn-kix_list_1-2;
    }
    ol.lst-kix_list_2-2.start {
        counter-reset: lst-ctn-kix_list_2-2 0;
    }
    .lst-kix_list_1-5 > li {
        counter-increment: lst-ctn-kix_list_1-5;
    }
    .lst-kix_list_1-8 > li {
        counter-increment: lst-ctn-kix_list_1-8;
    }
    ol.lst-kix_list_1-4.start {
        counter-reset: lst-ctn-kix_list_1-4 0;
    }
    ol.lst-kix_list_1-1.start {
        counter-reset: lst-ctn-kix_list_1-1 0;
    }
    ol.lst-kix_list_2-2 {
        list-style-type: none;
    }
    ol.lst-kix_list_2-3 {
        list-style-type: none;
    }
    ol.lst-kix_list_2-4 {
        list-style-type: none;
    }
    ol.lst-kix_list_2-5 {
        list-style-type: none;
    }
    .lst-kix_list_1-4 > li {
        counter-increment: lst-ctn-kix_list_1-4;
    }
    ol.lst-kix_list_2-0 {
        list-style-type: none;
    }
    .lst-kix_list_2-4 > li {
        counter-increment: lst-ctn-kix_list_2-4;
    }
    ol.lst-kix_list_1-6.start {
        counter-reset: lst-ctn-kix_list_1-6 0;
    }
    ol.lst-kix_list_2-1 {
        list-style-type: none;
    }
    ol.lst-kix_list_1-3.start {
        counter-reset: lst-ctn-kix_list_1-3 0;
    }
    ol.lst-kix_list_2-8.start {
        counter-reset: lst-ctn-kix_list_2-8 0;
    }
    ol.lst-kix_list_1-2.start {
        counter-reset: lst-ctn-kix_list_1-2 0;
    }
    .lst-kix_list_1-0 > li:before {
        content: '' counter(lst-ctn-kix_list_1-0, decimal) '. ';
    }
    ol.lst-kix_list_2-6 {
        list-style-type: none;
    }
    .lst-kix_list_1-1 > li:before {
        content: '' counter(lst-ctn-kix_list_1-1, lower-latin) '. ';
    }
    .lst-kix_list_1-2 > li:before {
        content: '' counter(lst-ctn-kix_list_1-2, lower-roman) '. ';
    }
    ol.lst-kix_list_2-0.start {
        counter-reset: lst-ctn-kix_list_2-0 0;
    }
    ol.lst-kix_list_2-7 {
        list-style-type: none;
    }
    ol.lst-kix_list_2-8 {
        list-style-type: none;
    }
    .lst-kix_list_1-3 > li:before {
        content: '' counter(lst-ctn-kix_list_1-3, decimal) '. ';
    }
    .lst-kix_list_1-4 > li:before {
        content: '' counter(lst-ctn-kix_list_1-4, lower-latin) '. ';
    }
    ol.lst-kix_list_1-0.start {
        counter-reset: lst-ctn-kix_list_1-0 0;
    }
    .lst-kix_list_1-0 > li {
        counter-increment: lst-ctn-kix_list_1-0;
    }
    .lst-kix_list_1-6 > li {
        counter-increment: lst-ctn-kix_list_1-6;
    }
    .lst-kix_list_1-7 > li:before {
        content: '' counter(lst-ctn-kix_list_1-7, lower-latin) '. ';
    }
    ol.lst-kix_list_2-7.start {
        counter-reset: lst-ctn-kix_list_2-7 0;
    }
    .lst-kix_list_1-3 > li {
        counter-increment: lst-ctn-kix_list_1-3;
    }
    .lst-kix_list_1-5 > li:before {
        content: '' counter(lst-ctn-kix_list_1-5, lower-roman) '. ';
    }
    .lst-kix_list_1-6 > li:before {
        content: '' counter(lst-ctn-kix_list_1-6, decimal) '. ';
    }
    li.li-bullet-0:before {
        margin-left: -18pt;
        white-space: nowrap;
        display: inline-block;
        min-width: 18pt;
    }
    .lst-kix_list_2-0 > li:before {
        content: '' counter(lst-ctn-kix_list_2-0, decimal) '. ';
    }
    .lst-kix_list_2-1 > li:before {
        content: '' counter(lst-ctn-kix_list_2-1, lower-latin) '. ';
    }
    ol.lst-kix_list_2-1.start {
        counter-reset: lst-ctn-kix_list_2-1 0;
    }
    .lst-kix_list_2-5 > li {
        counter-increment: lst-ctn-kix_list_2-5;
    }
    .lst-kix_list_2-8 > li {
        counter-increment: lst-ctn-kix_list_2-8;
    }
    .lst-kix_list_1-8 > li:before {
        content: '' counter(lst-ctn-kix_list_1-8, lower-roman) '. ';
    }
    .lst-kix_list_2-2 > li:before {
        content: '' counter(lst-ctn-kix_list_2-2, lower-roman) '. ';
    }
    .lst-kix_list_2-3 > li:before {
        content: '' counter(lst-ctn-kix_list_2-3, decimal) '. ';
    }
    .lst-kix_list_2-2 > li {
        counter-increment: lst-ctn-kix_list_2-2;
    }
    ol.lst-kix_list_2-4.start {
        counter-reset: lst-ctn-kix_list_2-4 0;
    }
    ol {
        margin: 0;
        padding: 0;
    }
    table td,
    table th {
        padding: 0;
    }
    .c2 {
        color: #000000;
        font-weight: 700;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 20pt;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c8 {
        color: #000000;
        font-weight: 400;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 11pt;
        font-family: 'Arial';
        font-style: normal;
    }
    .c1 {
        color: #000000;
        font-weight: 400;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 12pt;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c7 {
        color: #000000;
        font-weight: 700;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 12pt;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c9 {
        color: #000000;
        font-weight: 700;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 11pt;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c25 {
        color: #000000;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 16pt;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c6 {
        padding-top: 0pt;
        padding-bottom: 8pt;
        line-height: 1.0791666666666666;
        orphans: 2;
        widows: 2;
        text-align: center;
    }
    .c4 {
        padding-top: 0pt;
        padding-bottom: 6pt;
        line-height: 1.05;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    .c18 {
        padding-top: 0pt;
        padding-bottom: 0pt;
        line-height: 1;
        orphans: 2;
        widows: 2;
        text-align: center;
    }
    .c31 {
        font-weight: 400;
        text-decoration: none;
        vertical-align: baseline;
        font-size: 11pt;
        font-family: 'Trebuchet MS';
        font-style: normal;
    }
    .c19 {
        padding-top: 0pt;
        padding-bottom: 6pt;
        line-height: 1.0999999999999999;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    .c27 {
        font-weight: 400;
        text-decoration: none;
        vertical-align: baseline;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c0 {
        text-decoration-skip-ink: none;
        font-size: 12pt;
        -webkit-text-decoration-skip: none;
        color: #1155cc;
        text-decoration: underline;
    }
    .c29 {
        color: #000000;
        text-decoration: none;
        vertical-align: baseline;
        font-family: 'Calibri';
        font-style: normal;
    }
    .c17 {
        text-decoration-skip-ink: none;
        -webkit-text-decoration-skip: none;
        color: #0563c1;
        text-decoration: underline;
    }
    .c5 {
        padding-top: 0pt;
        padding-bottom: 0pt;
        line-height: 1;
        text-align: left;
    }
    .c33 {
        text-decoration: none;
        vertical-align: baseline;
        font-family: 'Calibri';
    }
    .c38 {
        -webkit-text-decoration-skip: none;
        text-decoration: underline;
        text-decoration-skip-ink: none;
    }
    .c12 {
        color: inherit;
        text-decoration: inherit;
    }
    .c32 {
        padding: 0;
        margin: 0;
    }
    .c20 {
        font-size: 15pt;
        color: #ff0000;
    }
    .c36 {
        /*max-width: 468pt;*/
        /*padding: 72pt 72pt 72pt 72pt;*/
    }
    .c14 {
        color: #980000;
        font-weight: 700;
    }
    .c28 {
        color: #2f5496;
        font-size: 11pt;
    }
    .c23 {
        margin-left: 36pt;
        padding-left: 0pt;
    }
    .c13 {
        border: 1px solid black;
        margin: 5px;
    }
    .c3 {
        font-size: 20pt;
        font-weight: 700;
    }
    .c39 {
        color: #2f5496;
    }
    .c10 {
        font-size: 12pt;
    }
    .c11 {
        height: 11pt;
    }
    .c16 {
        font-style: italic;
    }
    .c34 {
        color: #980000;
    }
    .c37 {
        font-size: 11pt;
    }
    .c24 {
        background-color: #ffffff;
    }
    .c21 {
        color: #ff0000;
    }
    .c26 {
        color: #444444;
    }
    .c22 {
        font-size: 15pt;
    }
    .c30 {
        color: #000000;
    }
    .c35 {
        font-size: 16pt;
    }
    .c15 {
        font-weight: 700;
    }
    .title {
        padding-top: 24pt;
        color: #000000;
        font-weight: 700;
        font-size: 36pt;
        padding-bottom: 6pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    .subtitle {
        padding-top: 18pt;
        color: #666666;
        font-size: 24pt;
        padding-bottom: 4pt;
        font-family: 'Georgia';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        font-style: italic;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    li {
        color: #000000;
        font-size: 11pt;
        font-family: 'Calibri';
    }
    p {
        margin: 0;
        color: #000000;
        font-size: 11pt;
        font-family: 'Calibri';
    }
    h1 {
        padding-top: 24pt;
        color: #000000;
        font-weight: 700;
        font-size: 24pt;
        padding-bottom: 6pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    h2 {
        padding-top: 18pt;
        color: #000000;
        font-weight: 700;
        font-size: 18pt;
        padding-bottom: 4pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    h3 {
        padding-top: 14pt;
        color: #000000;
        font-weight: 700;
        font-size: 14pt;
        padding-bottom: 4pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    h4 {
        padding-top: 12pt;
        color: #000000;
        font-weight: 700;
        font-size: 12pt;
        padding-bottom: 2pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    h5 {
        padding-top: 11pt;
        color: #000000;
        font-weight: 700;
        font-size: 11pt;
        padding-bottom: 2pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
    h6 {
        padding-top: 10pt;
        color: #000000;
        font-weight: 700;
        font-size: 10pt;
        padding-bottom: 2pt;
        font-family: 'Calibri';
        line-height: 1.0791666666666666;
        page-break-after: avoid;
        orphans: 2;
        widows: 2;
        text-align: left;
    }
</style>
<body class="c24 c36 doc-content">
    <p class="c6">
        <span class="c3">In-Person</span><span class="c3">&nbsp;</span
        ><span class="c3">HTAN</span><span class="c2">&nbsp;Data Jamboree</span>
    </p>
    <p class="c18"><span class="c15 c25">NIH Campus</span></p>
    <p class="c18"><span class="c25 c15">Bethesda, MD</span></p>
    <p class="c6"><span class="c15 c35">November 6-8, 2024</span></p>
    <p class="c6">
        <span class="c15 c22"
            >Participation Application Now Open: submit your application by </span
        ><span class="c15 c20">September 6, 2024</span>
    </p>
    <p class="c4 c11"><span class="c1"></span></p>
    <p class="c4 c24">
        <span class="c10"
            >The NIH&rsquo;s National Cancer Institute is pleased to announce an
            in-person Human Tumor Atlas Network (HTAN) Data Jamboree to promote
            the reuse of </span
        ><span class="c17 c10"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://data.humantumoratlas.org/&amp;sa=D&amp;source=editors&amp;ust=1723644802021862&amp;usg=AOvVaw3xrQHDfBbuG0tUMQvLHfXF"
                >HTAN atlas datasets</a
            ></span
        ><span class="c10 c26">&nbsp;with a focus </span
        ><span class="c10"
            >on spatial omics datasets. HTAN data is available through the </span
        ><span class="c17 c10"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://humantumoratlas.org/explore&amp;sa=D&amp;source=editors&amp;ust=1723644802022226&amp;usg=AOvVaw0M4hZIXQ7sOc9s0LcI9u18"
                >HTAN Data Portal</a
            ></span
        ><span class="c10 c26">&nbsp;</span
        ><span class="c10">and NCI Cancer Research Data Commons (CRDC)</span
        ><span class="c10 c26">&nbsp;</span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://datacommons.cancer.gov/explore/cloud-resources&amp;sa=D&amp;source=editors&amp;ust=1723644802022464&amp;usg=AOvVaw11_uZNeKz13V8p6MZ3WFmK"
                >Cloud Resources</a
            ></span
        ><span class="c10 c26 c27">.</span>
    </p>
    <p class="c4 c24">
        <span class="c10">Additional data resources include a </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://github.com/isb-cgc/Community-Notebooks/blob/master/HTAN/Python%2520Notebooks/A_Guide_to_HTAN_Data.ipynb&amp;sa=D&amp;source=editors&amp;ust=1723644802022767&amp;usg=AOvVaw3-fxft-44D4oVj9-prtfHb"
                >Guide to HTAN Data</a
            ></span
        ><span class="c10">&nbsp;and other </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/HTANNotebooks.html&amp;sa=D&amp;source=editors&amp;ust=1723644802022995&amp;usg=AOvVaw1CYcK3_fir6hVgt1ZzP3yQ"
                >HTAN</a
            ></span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/HTANNotebooks.html&amp;sa=D&amp;source=editors&amp;ust=1723644802023228&amp;usg=AOvVaw1jXiihd1ypLte4jEfVyIoU"
                >&nbsp;Notebooks</a
            ></span
        ><span class="c10">.</span>
    </p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4">
        <span class="c10 c15">To Apply: </span
        ><span class="c10">Please&nbsp;submit your application by </span
        ><span class="c10 c21 c15">September 6</span
        ><span class="c10 c21">&nbsp;</span><span class="c10">using the </span
        ><span class="c10"
            ><a
                class="c0"
                href="https://forms.gle/zJ8npSeeFgdjgYz97"
                target="_blank"
                >Application Form</a
            ></span
        ><span class="c39 c10">.</span>
    </p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4">
        <span class="c7">The Opportunity and a Call to Action</span>
    </p>
    <p class="c4">
        <span class="c10"
            >Spatial omics and single-cell technologies are becoming widely used
            across biomedical research, including in cancer. These rich and
            complex datasets provide an untapped resource for reuse to validate
            previous findings, perform new analyses, and </span
        ><span class="c10">integrate</span
        ><span class="c10"
            >&nbsp;with other datasets. However, the best approaches and tools
            for reusing these datasets are not </span
        ><span class="c10">well-defined</span><span class="c10">. </span
        ><span class="c10 c16 c30 c15 c33"
            >Do you have an interest in utilizing spatial omics and single-cell
            approaches in your cancer research? Do you have creative approaches
            for analyzing or integrating these data? Do you want to help improve
            the utility and accessibility of spatial omics and single-cell
            data?</span
        >
    </p>
    <p class="c4">
        <span class="c10"
            >Join colleagues from a wide range of fields and communities to
            explore new approaches to analyze and integrate </span
        ><span class="c17 c10"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://humantumoratlas.org/explore&amp;sa=D&amp;source=editors&amp;ust=1723644802024282&amp;usg=AOvVaw2dtTmKmutWqP8hveGnmlHd"
                >single-cell and spatial omics data from the Human Tumor Atlas
                Network (HTAN)</a
            ></span
        ><span class="c10"
            >. By participating in this event, you will have the opportunity to
            learn more about the HTAN data and related resources, help improve
            the utility of HTAN data, and help build a broader and more diverse </span
        ><span class="c10">HTAN</span><span class="c1">&nbsp;community.</span>
    </p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4"><span class="c7">About HTAN</span></p>
    <p class="c4">
        <span class="c10">The goal of the </span
        ><span class="c17 c10"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://humantumoratlas.org/&amp;sa=D&amp;source=editors&amp;ust=1723644802024830&amp;usg=AOvVaw2HDQPAVs9AgI8baQVZXJG1"
                >Human Tumor Atlas Network</a
            ></span
        ><span class="c10"
            >&nbsp;(HTAN) is to generate molecularly defined, multi-dimensional
            spatial atlases of tumors as they progress from </span
        ><span class="c10">precancer</span
        ><span class="c10"
            >&nbsp;to cancer, transition from locally invasive to metastatic
            disease, and respond or become resistant to therapy. A brief
            overview of HTAN can be seen </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://videocast.nih.gov/watch%3D45619&amp;sa=D&amp;source=editors&amp;ust=1723644802025114&amp;usg=AOvVaw1JUy6NXEvtcunMzAwvsFOZ"
                >here</a
            ></span
        ><span class="c1"
            >&nbsp;(starting at 02h:11m:56s). The initial set of HTAN atlases
            provide several new insights into the biology of precancer,
            metastasis, and therapy resistance and suggest opportunities for
            translation of several HTAN findings.
        </span>
    </p>
    <p class="c4">
        <span class="c10"
            >Importantly, HTAN also operates as a community resource and
            extensive efforts have been made across HTAN to provide the data, </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://www.protocols.io/workspaces/ncihtan&amp;sa=D&amp;source=editors&amp;ust=1723644802025399&amp;usg=AOvVaw3RNzvAWw2XZXEPeSctr6mr"
                >protocols</a
            ></span
        ><span class="c10">, and </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://humantumoratlas.org/tools&amp;sa=D&amp;source=editors&amp;ust=1723644802025615&amp;usg=AOvVaw0vZeWzluHAlGkQ4gFpQWOh"
                >analytical methods</a
            ></span
        ><span class="c10"
            >&nbsp;underlying the HTAN atlases available to the community. You
            can explore the available HTAN data across multiple precancer and
            cancer types, including breast, colon, lung, skin, pancreas, and
            pediatric leukemia, </span
        ><span class="c10 c17"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://humantumoratlas.org/explore&amp;sa=D&amp;source=editors&amp;ust=1723644802025801&amp;usg=AOvVaw0q0Pja9Kv4NEnpqn-1LxaH"
                >here</a
            ></span
        ><span class="c1">.</span>
    </p>
    <p class="c4 c11"><span class="c1"></span></p>
    <p class="c4"><span class="c10 c15">About the HTAN Data Jamborees</span></p>
    <p class="c4">
        <span class="c1"
            >Data Jamborees provide amazing opportunities for scientists and
            coders of diverse expertise and experience levels to work together
            creatively to build unique solutions that solve relevant problems in
            cancer research. &nbsp;The key to our Data Jamboree is the focus on
            interdisciplinary collaboration, fostering new ideas and insights,
            and expanding the HTAN community. Over three days, teams of four to
            eight people work on different scientific problems, emphasizing
            sharing expertise and insight through shared presentations and live
            demonstrations throughout the event. The event will wrap up with
            summary presentations or demonstrations of the work accomplished
            during the event.</span
        >
    </p>
    <p class="c4">
        <span class="c10"
            >Participants may leverage the NCI CRDC resources. </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://datacommons.cancer.gov/explore/cloud-resources&amp;sa=D&amp;source=editors&amp;ust=1723644802026340&amp;usg=AOvVaw0w-fGfdvrRd9mis23PJcx4"
                >CRDC cloud resources</a
            ></span
        ><span class="c10"
            >&nbsp;can provide an environment to work with HTAN data or combine
            HTAN data with publicly available datasets, including those
            generated by Jamboree participants. Additional data resources
            include a </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://github.com/isb-cgc/Community-Notebooks/blob/master/HTAN/Python%2520Notebooks/A_Guide_to_HTAN_Data.ipynb&amp;sa=D&amp;source=editors&amp;ust=1723644802026582&amp;usg=AOvVaw1r_niFGo_y8dJhkgdSCriY"
                >Guide to HTAN Data</a
            ></span
        ><span class="c10">&nbsp;and other </span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/HTANNotebooks.html&amp;sa=D&amp;source=editors&amp;ust=1723644802026797&amp;usg=AOvVaw1aOIabQ45O9xzb7AKFhsuk"
                >HTAN</a
            ></span
        ><span class="c0"
            ><a
                class="c12"
                href="https://www.google.com/url?q=https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/HTANNotebooks.html&amp;sa=D&amp;source=editors&amp;ust=1723644802027001&amp;usg=AOvVaw3LezSev6zFnMPJ00bJiKZt"
                >&nbsp;Notebooks</a
            ></span
        ><span class="c10">. </span
        ><span class="c10"
            >The HTAN atlas datasets contain clinical data matched to a range of
            molecular data types, including a variety of multiplexed spatial
            data types (i.e., multiplexed immunofluorescence and spatial
            transcriptomics) and bulk and single-cell omics data.
        </span>
    </p>
    <p class="c4">
        <span class="c10"
            >The HTAN Data Jamborees aim to bring together investigators from
            within and outside of HTAN </span
        ><span class="c10">who are interested</span
        ><span class="c10"
            >&nbsp;in utilizing HTAN data and associated tools. The Jamboree
            planning team will organize participants into small teams with
            diverse expertise and experience. </span
        ><span class="c10 c16 c15"
            >Each team will be focused on addressing a specific question or
            challenge utilizing available HTAN data.</span
        ><span class="c10">&nbsp;</span
        ><span class="c10 c16 c15 c38"
            >The fall 2024 Jamboree will have a focus on projects utilizing
            spatial omics data</span
        ><span class="c10 c16 c15">.</span
        ><span class="c10">&nbsp;Teams will work together </span
        ><span class="c10">in person</span
        ><span class="c1"
            >&nbsp;over three full days to develop creative solutions leveraging
            HTAN data and resources. The goals of the HTAN Data Jamborees
            include:</span
        >
    </p>
    <ol class="c32 lst-kix_list_1-0 start" start="1">
        <li class="c4 c23 li-bullet-0">
            <span class="c10 c30">Promote the </span
            ><span class="c17 c10"
                ><a
                    class="c12"
                    href="https://www.google.com/url?q=https://www.nature.com/articles/sdata201618&amp;sa=D&amp;source=editors&amp;ust=1723644802027615&amp;usg=AOvVaw12jA1ERNITnAIWpvxb70sc"
                    >FAIR</a
                ></span
            ><span class="c1">&nbsp;access and reuse of HTAN data</span>
        </li>
        <li class="c4 c23 li-bullet-0">
            <span class="c1"
                >Promote collaborations to expand the size and diversity of the
                HTAN community</span
            >
        </li>
        <li class="c4 c23 li-bullet-0">
            <span class="c10 c30">Promote </span><span class="c10">the </span
            ><span class="c1"
                >development of new methods and tools for HTAN data
                analysis</span
            >
        </li>
        <li class="c4 c23 li-bullet-0">
            <span class="c1"
                >Identify gaps and limitations of existing HTAN data and
                resources</span
            >
        </li>
    </ol>
    <p class="c4 c11"><span class="c1"></span></p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4"><span class="c10 c15">HTAN Data Jamboree Projects</span></p>
    <p class="c4">
        <span class="c10 c15 c16">Transcription Program Gradients: </span
        ><span class="c1"
            >This project aims to identify and interpret spatial gradients of
            gene programs in tumor spatial transcriptomic datasets.</span
        >
    </p>
    <p class="c4">
        <span class="c10 c16 c15">Validation of CAF Subtypes:</span
        ><span class="c10 c15">&nbsp;</span
        ><span class="c1"
            >This project will focus on the reproducibility of cancer associated
            fibroblast (CAF) functional subtype classification described in L.
            Cords et al, 2023, using spatial transcriptomic data for gene
            signature analysis and corresponding images for detection of cell
            morphology and spatial patterns with respect to cell type
            neighborhoods or anatomical structures.</span
        >
    </p>
    <p class="c4">
        <span class="c10 c16 c15"
            >Deep Learning on Multiplexed Immunofluorescence: </span
        ><span class="c1"
            >This project entails modeling multiplexed immunofluorescence
            imaging data with self-supervised deep learning to predict patient
            outcomes, identify cellular niches, and uncover the biology
            informing the model predictions with feature attribution
            methods</span
        >
    </p>
    <p class="c4">
        <span class="c10 c16 c15"
            >Analyzing Cancer Spatial Transcriptomic Data with spatiAlytica LLM: </span
        ><span class="c1"
            >This project will extend spatiAlytica to be a cancer-specific
            multi-agent system by identifying cancer-related questions and
            developing corresponding Python code, creating cancer-specific
            analysis actions, generating a benchmark cancer spatial
            transcriptomic datasets, and evaluating spatiAlytica&#39;s
            capabilities with the benchmark dataset.</span
        >
    </p>
    <p class="c4">
        <span class="c10 c16 c15"
            >Heterogeneity, Ancestry, and Tumor Progression: </span
        ><span class="c1"
            >This project will use a multi-Omics spatial approach to unravel the
            complexity of tumor heterogeneity and understand disease progression
            influenced by genetic ancestry and/or Self-Reported Race, as well as
            drive poor outcomes among underrepresented populations</span
        >
    </p>
    <p class="c4">
        <span class="c10 c16 c15">Pan-cancer Treg Atlas:</span
        ><span class="c10"
            >&nbsp;This project will construct a pan-cancer regulatory T cell
            (Treg) atlas using scRNA-seq data, and validate Treg subtypes and
            function using spatial transcriptomic data.</span
        >
    </p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4"><span class="c7">Who Should Apply</span></p>
    <p class="c4">
        <span class="c10"
            >Independent investigators, postdocs, and late-stage graduate
            students interested in utilizing HTAN spatial omics and single-cell
            data as part of their research.</span
        ><span class="c10"
            >&nbsp;Investigators with cancer biology, cancer clinical research, </span
        ><span class="c10">pathology</span
        ><span class="c10"
            >, imaging biology, systems biology, computational biology, data
            science, bioinformatics and other relevant fields are all encouraged
            to apply. </span
        ><span class="c10 c14"
            >Computational expertise is not required to participate</span
        ><span class="c10 c34">.</span
        ><span class="c1"
            >&nbsp;Participants will be organized into teams with diverse
            expertise by the organizing committee.</span
        >
    </p>
    <p class="c4 c11"><span class="c9"></span></p>
    <p class="c4"><span class="c7">How To Apply</span></p>
    <p class="c4">
        <span class="c10">Please&nbsp;submit your application by </span
        ><span class="c10 c21 c15">September 6</span
        ><span class="c10 c21">&nbsp;</span><span class="c10">using the </span
        ><span class="c10"
            ><a
                class="c0"
                href="https://forms.gle/zJ8npSeeFgdjgYz97"
                target="_blank"
                >Application Form</a
            ></span
        ><span class="c10 c39">.</span>
    </p>
    <p class="c4 c11"><span class="c7"></span></p>
    <p class="c4"><span class="c7">Questions?</span></p>
    <p class="c4">
        <span class="c10"
            >For questions about the HTAN Data Jamborees in general or about the
            application process, please email Sean Hanlon (</span
        ><span class="c0"
            ><a class="c12" href="mailto:sean.hanlon@nih.gov"
                >sean.hanlon@nih.gov</a
            ></span
        ><span class="c10">)</span>
    </p>
</body>
